src/lib/ApxCore/App/AppNotifyListModel.h

```diff
@@ -66,7 +66,10 @@ class AppNotifyListModel : public QAbstractListModel
     };
     QList<NotifyListItem *> m_items;
 
-protected:
+private slots:
+    void notification(QString msg, QString subsystem, AppNotify::NotifyFlags flags, Fact *fact);
```

Comment:
redundant `notification` method

Reply:
> redundant `notification` method

The provided code is not part of a modification but rather pre-existing code. It got into commit as a result of refactoring the order of declaring signals and slots. Its redundancy was not investigated within the scope of the task. Considering the new information about its origin, should it indeed be considered "redundant"